## file 'Hint.msg'

# the type of hint from the oracle
string HintType
# the content of the hint
string HintContent
